packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/SocialLoginCard.js

```diff
@@ -1,21 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import LoginCard from './LoginCard';
+import { excludeKeys } from '../../../../common/helpers';
 
 const SocialLoginCard = ({ children, className, ...props }) => (
-  <LoginCard {...props} className={`login-pf-accounts ${className}`}>
+  <LoginCard {...excludeKeys(props, ['className'])} className={classNames('login-pf-accounts', className)}>
```

Comment:
request:
declaring `className` should make `excludeKeys` unnecessary, I think you had this before:
```
<LoginCard {...props} className={classNames('login-pf-accounts', className)}>
``` 
that should be good to go